ruby/vampires.rb

```diff
@@ -0,0 +1,83 @@
+insurance = ''
```

Comment:
In ruby we don't need to set variables to default values, you can just set them once when you need to